src/jaeger.rs

```diff
@@ -0,0 +1,37 @@
+// Originally retired from davidB/tracing-opentelemetry-instrumentation-sdk
+// which is licensed under CC0 1.0 Universal
+// https://github.com/davidB/tracing-opentelemetry-instrumentation-sdk/blob/d3609ac2cc699d3a24fbf89754053cc8e938e3bf/LICENSE
+
+use opentelemetry::trace::TraceError;
+use opentelemetry_jaeger::config::agent::AgentPipeline;
+use opentelemetry_sdk::{
+    trace::{config, Sampler, Tracer},
+    Resource,
+};
+use opentelemetry_semantic_conventions as semcov;
+
+#[must_use]
+pub fn identity(v: AgentPipeline) -> AgentPipeline {
```

Comment:
Shouldn't this generate some warnings due to this function not being used anywhere?

Reply:
It seems like the original idea was to generate a warning if the corresponding feature is set, but not used. But it doesn't work like this for some reason.